lib/registerPlugins.js

```diff
@@ -41,7 +41,10 @@ function registerCustomPlugin(server, callback) {
     async.eachSeries(['shutdown'], (pluginName, next) => {
         server.register({
             register: require(`../plugins/${pluginName}`),
-            options: { terminationGracePeriod: process.env.TERMINATION_GRACE_PERIOD || 30 }
+            options: {
+                terminationGracePeriod: parseInt(process.env.TERMINATION_GRACE_PERIOD, 2)
```

Comment:
radix should be 10. Alternatively you can just do `+process.env.TERMINATION_GRACE_PERIOD`

Reply:
fixed

Reply:
why not just Number?

Reply:
both provide same outcome. https://stackoverflow.com/questions/4090518/what-is-the-difference-between-parseint-and-number

`+` is same as Number()